Sources/LiveViewNative/Modifiers/Drawing and Graphics Modifiers/ClippedModifier.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  ClippedModifier.swift
+//  LiveViewNative
+//
+//  Created by Dylan.Ginsburg on 4/19/23.
+//
+
+import SwiftUI
+
+/// Clips this view to its bounding rectangular frame.
+///
+/// ```html
+/// <Text modifiers={clipped(@native, antialiased: true)}>Hello, world!</Text>
```

Comment:
The modifier looks good, but can you give an example where the effect will be visible? Maybe sticking an explicit frame on the Text before the clipped modifier